### PR TITLE
replace "system" language

### LINKED ISSUE #1603: Change "system" to QPU (quantum processing unit)
Except when referring to Systems One and Two, replace "system" with QPU

### PR DESCRIPTION
**Pull request by abbycross**
Closes #1603 

Use language other than "system". The following potentially require updating, as a result of my ripgrep search:

- [x] support.mdx
- [x] guides/estimate-job-run-time.mdx
~- [ ] guides/transpiler-plugins.ipynb ecosystem only~
- [x] guides/execution-modes-faq.mdx
- [x] guides/hello-world.ipynb
- [x] guides/qpu-information.mdx
- [x] guides/local-testing-mode.ipynb
~- [ ] guides/synthesize-unitary-operators.mdx physics sense only~
- [x] guides/manage-cost.mdx
- [x] guides/get-started-with-primitives.mdx
- [x] guides/max-execution-time.mdx
- [x] guides/qiskit-serverless.mdx
- [x] guides/configure-error-mitigation.mdx
- [x] guides/visualize-results.ipynb
- [x] guides/transpile.mdx
- [x] guides/fair-share-scheduler.mdx
- [x] guides/configure-error-suppression.mdx
- [x] guides/introduction-to-qasm.mdx
- [x] guides/debugging-tools.mdx
~- [ ] guides/build-noise-models.ipynb no changes needed~
- [x] guides/run-jobs-session.mdx
~- [ ] guides/install-qiskit-source.mdx no changes needed~
- [x] guides/plot-quantum-states.ipynb
- [x] guides/retired-qpus.mdx
- [x] guides/pulse.ipynb
- [x] guides/install-qiskit.mdx
- [x] guides/defaults-and-configuration-options.ipynb
~- [ ] guides/operators-overview.ipynb no changes needed~
~- [ ] guides/simulate-stabilizer-circuits.ipynb no changes needed~
- [x] guides/run-jobs-batch.mdx
- [x] guides/setup-channel.mdx
~- [ ] guides/operator-class.ipynb no changes needed~
~- [ ] guides/qasm-feature-table.mdx no changes needed~
- [x] guides/sessions.mdx
~- [ ] guides/create-transpiler-plugin.ipynb no changes needed~
~- [ ] guides/specify-observables-pauli.mdx no changes needed~
- [x] guides/simulate-with-qiskit-sdk-primitives.mdx
~- [ ] guides/custom-transpiler-pass.ipynb no changes needed~
- [x] guides/index.mdx
- [x] guides/instances.mdx
- [x] guides/processor-types.mdx
~- [ ] guides/configure-qiskit-local.mdx no changes needed~
- [x] guides/execution-modes.mdx
- [x] guides/native-gates.mdx
- [x] guides/transpiler-stages.ipynb
~- [ ] guides/circuit-library.ipynb no changes needed~
- [x] guides/simulate-with-qiskit-aer.ipynb
- [x] guides/repetition-rate-execution.mdx
- [x] guides/transpile-rest-api.mdx
- [x] guides/represent-quantum-computers.ipynb
- [x] guides/runtime-options-overview.mdx
- [x] guides/monitor-job.mdx
- [x] guides/dynamic-circuits-considerations.ipynb
- [x] guides/primitives.mdx (would like a review on this one)
- [x] guides/primitives-examples.mdx
~- [ ] start/configure-qiskit-local.mdx~

Migration guides:
- [x] qiskit-runtime.mdx
- [x] qiskit-1.0-installation.mdx
- [x] v2-primitives.mdx
- [x] qiskit-1.0-features.mdx
- [x] local-simulators.mdx
- [x] qiskit-quantum-instance.mdx (several outstanding questions)
- [x] sessions.mdx
- [x] qiskit-runtime-examples.mdx
- [x] qiskit-runtime-options.mdx
- [x] qiskit-runtime-use-case.mdx
- [x] qiskit-algorithms-module.mdx
- [x] qiskit-opflow-module.mdx

### CODE REVIEW

File: docs/guides/primitives.mdx [hunk 1]
```diff
@@ -288,11 +288,11 @@ These classes are reference implementations of both primitives and use the simul
...
 
 3. `BackendSampler` and `BackendEstimator`
 
-If a provider does not support primitives natively, you can use these classes to “wrap” any system into a primitive. Users can write primitive-style code for providers that don’t yet have a primitives-based interface. These classes can be used just like the regular Sampler and Estimator, except they should be initialized with an additional `backend` argument for selecting which system to run on.
+If a provider does not support primitives natively, you can use these classes to “wrap” any quantum computer into a primitive. Users can write primitive-style code for providers that don’t yet have a primitives-based interface. These classes can be used just like the regular Sampler and Estimator, except they should be initialized with an additional `backend` argument for selecting which quantum computer to run on.
```
Comment (kaelynj) [inline docs/guides/primitives.mdx:295]:
```suggestion
If a provider does not support primitives natively, you can use these classes to “wrap” any quantum computing resource into a primitive. Users can write primitive-style code for providers that don’t yet have a primitives-based interface. These classes can be used just like the regular Sampler and Estimator, except they should be initialized with an additional `backend` argument for selecting which quantum computer to run on.
```

File: docs/guides/primitives.mdx [hunk 2]
```diff
@@ -279,7 +279,7 @@ probabilities, except they might include negative values, which can occur when u
 ## How to use Qiskit primitives
 
 The `qiskit.primitives` module enables the development of primitive-style quantum programs and was specifically
-designed to simplify switching between different types of systems. The module provides three separate classes
+designed to simplify switching between different types of quantum computers. The module provides three separate classes
```
Comment (kaelynj) [inline docs/guides/primitives.mdx:282]:
```suggestion
designed to simplify switching between different types of quantum compute resources. The module provides three separate classes
```

File: docs/guides/primitives.mdx (hunk 1 above)
Comment (kaelynj) [reply docs/guides/primitives.mdx:295]: I think in these two places referring to the quantum computers as a "quantum compute resource" or "quantum computing resource" feels more appropriate.

**Review by beckykd — APPROVED** [review]
Dancing Banana!

**Pull request merged**